client-common/src/main/java/org/apache/cassandra/sidecar/common/ApiEndpointsV1.java

```diff
@@ -116,6 +116,10 @@ public final class ApiEndpointsV1
     public static final String ABORT_RESTORE_JOB_ROUTE = RESTORE_JOB_ROUTE + ABORT;
     public static final String RESTORE_JOB_PROGRESS_ROUTE = RESTORE_JOB_ROUTE + PROGRESS;
 
+    // Auth related APIs
+    public static final String CACHE_NAME_PARAM = ":cacheName";
+    public static final String INVALIDATE_CACHE_ROUTE = API_V1 + "/caches/" + CACHE_NAME_PARAM + "/invalidate";
```

Comment:
We should be able to invalidate with a specific key too. For e.g. If we want to invalidate a single identity in `identity_to_role_cache`